Traceback (most recent call last):
  File "/eos/user/z/zhenxuan/hhwwgg/GJet_Pt-40toInf_2017/job_105/GJet_Pt-40toInf_2017_executable_job105.py", line 13, in <module>
    run_analysis(config)
  File "/afs/cern.ch/user/z/zhenxuan/HiggsDNA/higgs_dna/analysis.py", line 77, in run_analysis
    tag_sequence = TagSequence(
  File "/afs/cern.ch/user/z/zhenxuan/HiggsDNA/higgs_dna/taggers/tag_sequence.py", line 50, in __init__
    m_tagger = self.create_tagger(tagger)
  File "/afs/cern.ch/user/z/zhenxuan/HiggsDNA/higgs_dna/taggers/tag_sequence.py", line 63, in create_tagger
    module = importlib.import_module(config["module_name"])
  File "/eos/user/z/zhenxuan/higgs-dna/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/afs/cern.ch/user/z/zhenxuan/HiggsDNA/higgs_dna/taggers/diphoton_tagger.py", line 14, in <module>
    from higgs_dna.selections import gen_selections
  File "/afs/cern.ch/user/z/zhenxuan/HiggsDNA/higgs_dna/selections/gen_selections.py", line 48, in <module>
    @numba.njit
NameError: name 'numba' is not defined
